tests/runtime/data/stackdriver/stackdriver_test_operation.h

```diff
@@ -0,0 +1,74 @@
+#define OPERATION_COMMON_CASE	"["		\
+	"1591111124,"			\
+	"{"				\
+        "\"logging.googleapis.com/operation\": "		\
+        "{"            \
+            "\"id\": \"test_id\","          \
+            "\"producer\": \"test_producer\","      \
+            "\"first\": true,"      \
+            "\"last\": true"       \
+        "}"     \
+	"}]"
+
+#define EMPTY_OPERATION	"["		\
+	"1591111124,"			\
+	"{"				\
+        "\"logging.googleapis.com/operation\": "		\
+        "{"            \
+        "}"     \
+	"}]"
+
+#define OPERATION_IN_STRING "["		\
+	"1591111124,"			\
+	"{"				\
+    "\"logging.googleapis.com/operation\": \"some string\""		\
+	"}]"
+
+#define EMPTY_SUBFIELDS	"["		\
```

Comment:
The subfields are default values of the corresponding type, not empty. I don't think we need to test default values specifically.

I think the partial_subfileds tests below cover this (which also means some subfields are empty in jsonPayload so the code will populate the default values in logEntry for these subfields).

What do you think?

Reply:
Updated. I agree with you:)